po/es.po

```diff
@@ -8,18 +8,18 @@ msgstr ""
 "Project-Id-Version: com.github.sgpthomas.hourglass\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-05-03 10:10+0200\n"
-"PO-Revision-Date: 2020-05-01 16:18+0200\n"
+"PO-Revision-Date: 2020-05-28 16:47-0500\n"
 "Last-Translator: Sam Thomas <sgpthomas@gmail.com>\n"
 "Language-Team: none\n"
 "Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.0.6\n"
```

Comment:
```suggestion
```

I prefer to remove this line because this line is added to git history as an extra changes when the version of Poedit increases.